Deeploy/Targets/Generic/Parsers.py

```
@@ -3124,3 +3215,236 @@ class GlobalMaxPoolParser(GlobalPoolParser):
+        image_shape = self.operatorRepresentation['image_shape']
+        block_shape = self.operatorRepresentation['block_shape']
+        col_dims = self.operatorRepresentation['col_dims']
+
+        N, C = data_out.shape[0], data_out.shape[1]
+        block_volume = int(np.prod(block_shape))
+        L = int(np.prod(col_dims))
+
+        if list(data_in.shape) != [N, C * block_volume, L]:
+            return ctxt, False
+
+        if list(data_out.shape) != [N, C] + image_shape:
```

Ruff (0.15.20)

[warning] Consider `[N, C, *image_shape]` instead of concatenation

Replace with `[N, C, *image_shape]`

(RUF005)